From bd4e2c5b0a11f09502f9bb9a6d842019a2fb7705 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Wed, 7 Dec 2016 12:06:58 +0200
Subject: [PATCH] update-kernel: use busybox's losetup always

fixes #6517

The output format of util-linux losetup is different, and there's
no easy way to get uniform output from these two implementations.
---
 update-kernel.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/update-kernel.in b/update-kernel.in
index cd1edc0..88f266b 100644
--- a/update-kernel.in
+++ b/update-kernel.in
@@ -133,7 +133,7 @@ if [ -z "$DESTDIR" ]; then
 	while read MOUNT; do
 		set -- $MOUNT
 		[ $2 = /.modloop ] || continue
-		DESTDIR=$(dirname $(losetup $1 | cut -d " " -f 3))
+		DESTDIR=$(dirname $(busybox losetup $1 | cut -d " " -f 3))
 		MNTDIR=$(dirname "$DESTDIR")
 		break
 	done < /proc/mounts
-- 
2.11.0

